code/datums/looping_sounds/looping_sound.dm

```diff
@@ -78,6 +80,12 @@
 /datum/looping_sound/proc/play(soundfile)
 	var/list/atoms_cache = output_atoms
 	var/sound/S = sound(soundfile)
+	if(area_sound)
+		for(var/area/sound_outputs in atoms_cache)
+			for(var/mob/listener in mobs_in_area(sound_outputs, TRUE) || 1)
```

Comment:
```suggestion
			for(var/mob/listener in mobs_in_area(sound_outputs, TRUE))
```
I doubt that one should be there. If the volume works without then you can leave out the `|| 1`